reference.md

```diff
@@ -293,7 +438,7 @@ These options are available for all types of interfaces.
 
      ``type`` (scalar)
      :    The type of route. Valid options are "unicast" (default),
-          "unreachable", "blackhole" or "prohibited".
+          "unreachable", "blackhole" or "prohibit".
```

Comment:
Should there be a serial comma here?

Reply:
@murraybd no, Canonical does not use serial commas: [style guide](https://wiki.canonical.com/Marketing/Style_guide#Serial_comma)

Reply:
We use a [checklist](https://canonical-webteam.github.io/practices/content/copy-reviews.html) that distils these rules in the webteam